src/components/LiveButton.ts

```diff
@@ -162,6 +162,8 @@ function isLive(player: ChromelessPlayer, threshold: number): boolean {
         const seekable = player.seekable;
         if (seekable.length > 0) {
             return seekable.end(seekable.length - 1) - player.currentTime <= threshold;
+        } else if (player.source?.streamType === 'live') {
+            return true;
```

Comment:
I don't think we populate `streamType` if it wasn't configured upfront when setting the source, or do we? If not we need to make sure it still works even if not configured. What do `duration`, `seekable` and `currentTime` look like when we're playing Millicast?

Reply:
@MattiasBuelens Ok, I like that better. The streamType indeed is not set by default. I was unsure if there was a use case where seekable would be empty and duration was infinity where the indicator would need to be false, since it was set up that way before. It makes sense to me that in that case it would always be true. 👍